### DISCUSSION TITLE
lavalink music not playing problem

### DISCUSSION THREAD

**Discussion by ibrahim4123** [lavalink-devs/Lavalink#745] [Q&A] [answered]
I set the email and password to disable youtube age in lavalink, but it creates token fln while playing the music and it doesn't play the music when entering the sound, how do I solve it?

**Comment by topi314** [comment]
can you please provide logs and further info?

**Comment by ibrahim4123** [comment]
![image](https://user-images.githubusercontent.com/83676691/190916354-6740b1f8-2688-4db0-9775-d60c8319f64f.png)
![image](https://user-images.githubusercontent.com/83676691/190916372-4f4cc450-5a10-4522-88ab-f916a8b2d4f8.png)
![image](https://user-images.githubusercontent.com/83676691/190916387-3298624a-958b-4aca-908d-2a79f031c680.png)
it says it plays like this but it doesn't play the song

**Comment by topi314** [comment, marked as answer]
there has been a lavaplayer fix for silence tracks. This might be related https://github.com/Walkyst/lavaplayer-fork/releases/tag/1.3.99
lavalink https://github.com/freyacodes/Lavalink/releases/tag/3.6.1 has this fix
